### DISCUSSION TITLE
Aggregate Device Setup on 2023 MBPro M2 MAX - Ventura 13.5.2

### DISCUSSION THREAD

**Discussion by harmonicalchemy** [ExistentialAudio/BlackHole#740] [General]
Hi Everyone.. I am using BlackHole 16ch with MOTU M6 H/W interface, Reaper, pure-data, & other digital audio apps, etc... 

In the Wiki under **Aggregate Device** / **4. Select Devices** 
It says: 

_MPORTANT: Due to issues with macOS the Built-in Output or Built-in Microphone must be enabled and listed as the top device in the Aggregate_

**This does NOT seem to be an issue for me...**

I configure my Aggregate Device with BlackHole 16ch as Clock Source (S/R: 48.0kHz) and it is listed at the TOP of the list.. All other devices: MBPro Speakers, microphone, headphones, etc. have Drift Correction ticked...  Using this configuration I have not experienced any glitches, latency issues, etc... even after running for days without rebooting etc...  It is ALL WORKING FANTASTIC with crystal clear sound... I LOVE BlackHole btw... I used to use JACK and that is real problematic on NEW Macs!

So.. I am Wondering if others _(on newer Apple silicon macs)_ are experiencing problems?

If NOT then maybe that note above _(in the Wiki)_ should be changed to reflect my and others problem free experience?

Also I am interested to know what exactly the supposed problems / symptoms are? _(i.e., a bit more detailed explanation within that note above)_

**THANKS!**

**Comment by devinroth** [comment]
Thanks for the compliments.

Perhaps they have fixed it. Maybe it’s only an issue with Multi-Outputs nows. Could you give it a test for me?

Definitely worth updating the text if that’s the case.

Devin

**Comment by harmonicalchemy** [comment]
Yes.. I will do what ever you request... The LEAST I can do for such a Nicely Built thing... (Swift? - my brother is a Swift dude)

Anyway tell me what to configure and what to look out for and I will do just that and report my findings...
Cheers,
Alisha

@harmonicalchemy (GitHub, Twitter, etc.)
PGP Fingerprints:(new 2020 keys)
Alisha Awen, 0x9D67B6DCE4F47517:
FF3E 38C9 E731 CCEE 17D6
3A92 9D67 B6DC E4F4 7517
git repo manager, 0xF7383D809BB271AC:
B36E B827 6720 E265 196C
20B2 F738 3D80 9BB2 71AC

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
